dotCMS/src/integration-test/java/com/dotcms/datagen/FileDescriptorDataGen.java

```diff
@@ -66,6 +67,7 @@ public FilterDescriptor next() {
 
```

Comment:
This class should be renamed to **FilterDescriptorDataGen** instead of **FileDescriptorDataGen**

Reply:
done https://github.com/dotCMS/core/pull/19955/commits/0026829b583a3c6a351de5e7f0dff3fab2c3187d